client/controller/index.web.js

```diff
@@ -60,7 +61,7 @@ export const makeLayout = makeLayoutMiddleware( ReduxWrappedLayout );
  * divs.
  */
 export function clientRouter( route, ...middlewares ) {
-	page( route, ...middlewares, render );
+	page( route, ...middlewares, render, makeLayout, clientRender );
```

Comment:
This shouldn't be codemodded -- `clientRouter` is our helper for isomorphic/SSR'd stuff. (If you look at the import, `render` is actually identical to `clientRender`, and `makeLayout` isn't needed here since we're expecting a single tree anyway.)